README.md

```diff
@@ -1,5 +1,18 @@
 # Software Engineering Team Design Project
 
+## Protoype
+[Click here to view Figma prototype](https://www.figma.com/file/1NeYMymzl5OYosLV0D24rM/PROTOTYPE)
+
+## Reports
+
```

Comment:
do we need a link for this ?

Reply:
a link for what?

Reply:
reports shouldnt we combine all of the docs into one report and have a link for it